lib/index.js

```diff
@@ -4,6 +4,32 @@
 var os = require('os');
 var _ = require('underscore');
 /**
+ * Default options
+ */
+var options;
+var defaultOptions = {
```

Comment:
You think it's best to keep **defaultOptions** here instead of inside **parseOptions** function?